owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1647,6 +1657,36 @@ class FileDisplayActivity : FileActivity(), FileFragment.ContainerActivity, OnEn
         manageOptionLockSelected(type)
     }
 
+    private fun manageDataIntent(uri: Uri?) {
+        if (uri != null && AccountUtils.getAccounts(applicationContext).isEmpty()) {
+            showMessageInToast(getString(R.string.no_account_configured))
+        } else if (uri != null && AccountUtils.getAccounts(applicationContext).size == 1) {
+            isFileDiscovered(uri).let { OCFile ->
```

Comment:
I would use lowerCamelCase for the name inside the `let` as we do with variable names

Reply:
Fixed